demo/fiber/oneshot_test/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
这个文件来源是？ 为什么没有copyright？

Reply:
我没有在loom里看到任何相关的copyright，之前跟江莎老师沟通过这个问题，他说那就不写copyright了。这个文件是我新建oneshot_test这个maven工程时自己写的，其中有些内容是intellij IDEA辅助生成。

Reply:
如果是自己写的，可以参考一下demo/fiber目录下其他pom.xml文件，加上tencent的copyright

Reply:
知道了，感谢老师的建议

Reply:
@Jong111 
如果是从Loom过来的文件，就保持原有的copyright。如果原来没有，也保持为空。
但如果是自己新创建的文件，那就加上腾讯的copyright声明。
```
Copyright (C) 2023 THL A29 Limited, a Tencent company. All rights reserved.
DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.

This code is free software; you can redistribute it and/or modify it
under the terms of the GNU General Public License version 2 only, as
published by the Free Software Foundation.

This code is distributed in the hope that it will be useful, but WITHOUT
ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
version 2 for more details (a copy is included in the LICENSE file that
accompanied this code).

You should have received a copy of the GNU General Public License version
2 along with this work; if not, write to the Free Software Foundation,
Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
```

Reply:
明白了，谢谢两位老师